test/system/700-play.bats

```diff
@@ -1035,3 +1035,37 @@ spec:
     run_podman rm -f -t 0 -a
     run_podman rmi automount_test
 }
+
+@test "podman kube restore user namespace" {
+    skip_if_remote "userns=auto is set on the server"
```

Comment:
I don't understand this comment? Why should a user not be able to to --userns auto on the remote client?

Reply:
that is the same condition we have in the "podman userns=auto in config file" test.  I think we do that because we have no way to verify if there are additional IDs configured on a different host

Reply:
Well we cannot run system tests on a different host anyway so this is not a concern.
This really should be tested on remote

Reply:
dropped